### DISCUSSION TITLE
Where can I find current state (like prompt) of my application in Postgres?

### DISCUSSION THREAD

**Discussion by MaxYaz** [langgenius/dify#8985] [General]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### Content

I have a pretty simple application in Dify. 
![Screenshot 2024-10-02 at 12 27 36](https://github.com/user-attachments/assets/20a4e01e-3de7-4401-8909-57d218e6686b)
And of course it has a prompt inside. Which table in Postgres contains the current state of the prompt?
My idea is to find a table where this prompt is stored and track where it is getting changed.

**Comment by y-omr** [comment]
Hey bro,
The prompt is probably stored in the "query" column of the "messages" table in the "Dify" database.
Additionally, other data such as "conversation_id," "answer," and "status" are also stored.
![image](https://github.com/user-attachments/assets/86f999cc-edbe-4c34-a0ea-af550dc72aaa)

**Reply by MaxYaz** [reply]
Hey!
It is actually in the workflows table inside graph column. It stores the state of application. So when ever you press update button new row gets created
